app/views/abuse_reports/new.html.erb

```diff
@@ -45,21 +55,24 @@
   </p>
 
   <ul>
-    <li><%= t(".include.username") %></li>
-    <li><%= t(".include.links") %></li>
-    <li><%= t(".include.quote_html",
-              tos_link: (link_to t(".include.ao3"), tos_path(anchor: "content"))) %></li>
-    <li><%= t(".include.sources") %></li>
+    <li><%= t(".include.username_html",
+              username_link: link_to(t(".include.username"), tos_faq_path(anchor: "user_unknown"))) %></li>
```

Comment:
I'm slightly worried that the variable, link text and overall sentence all starting with `username` may be confusing for translator (meaning it's unclear which one is the link text and which one is the sentence). Could `username_html` maybe be renamed? Best I can come up with is `include_username_html`.

Same goes for the `sources` point below, there the three variables also overlap in name.